numba_dpex/tests/dpjit_tests/dpnp/test_dpnp_empty.py

```diff
@@ -67,3 +91,19 @@ def func(shape):
     dummy_tensor = dpctl.tensor.empty(shape)
 
     assert c.dtype == dummy_tensor.dtype
+    assert c.usm_type == dummy_tensor.usm_type
+    assert c.sycl_device == dummy_tensor.sycl_device
+
+
+def test_dpnp_empty_exceptions():
+    queue = dpctl.SyclQueue()
+
+    @dpjit
+    def func2(a, q):
+        c = dpnp.empty_like(a, sycl_queue=q, device="cpu")
```

Comment:
Why `device="cpu"`?

Reply:
It is just a placeholder. The intent is to test if an exception is raised if both `sycl_queue` and `device` are specified.